internal/fileutils/path_helpers_lin_mac.go

```diff
@@ -0,0 +1,15 @@
+// +build !windows
+
+package fileutils
+
+// GetShortPathName returns the Windows short path
+// This function does not alter the path in any way on Linux or MacOS
+func GetShortPathName(path string) (string, error) {
```

Comment:
Nit: I think it would be cleaner if you made `ResolveUniquePath` be OS specific, and made these functions just things used only on the Windows implementation.